tests/Unit/Helpers/Evolution/Systems/Cce/AnalyticSolutions/AnalyticDataHelpers.cpp

```diff
@@ -42,6 +42,13 @@ extract_bondi_scalars_from_cartesian_metric(
       }
     }
   }
+  get<0, 0>(inverse_spherical_metric) =
```

Comment:
what are these changes for?  Should they be in another commit?

Reply:
These fix a mistake in confusing Boyer-lindquist time with retarded time that were caught by the ADM check. I think you're right that they should be in a separate commit.